README.md

```diff
@@ -2,6 +2,8 @@
 
 This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app), using the [Redux](https://redux.js.org/) and [Redux Toolkit](https://redux-toolkit.js.org/) template.
 
+The [React Loader Spinner](https://www.npmjs.com/package/react-loader-spinner) was implemented to render a spinner while API requests are being made. This spinner was created in a separate component, and can be imported and rendered from other components when necessary.
```

Comment:
Bien. Puede parecer una tontería, pero estaría bueno agregar, a pesar de que es obvio, cómo usar el Spinner en un componente, o sea, que va "<Spinner />" sin props. Siempre que documentamos algo, ponemos un ejemplo de cómo usarlo, aunque sea muy simple.

Reply:
I uploaded the requested changes here https://github.com/alkemyTech/OT104-Client/pull/56/commits/1717f6eef499562fdaf1d767634591fc3dcb0935.